account/templates/base.html

```diff
@@ -119,7 +119,7 @@
             <div class="span4">
                 <div class="well">
                 <h4>Login</h4>
-                    <form class="form" action="/account/login/?next={% url account.views.home %}"
+                    <form class="form" action="/account/login/?next=/"
```

Comment:
Should be `'home'` then, given the other changes which use the name now - and then it's OK to assume it to be `/` anyway.  So, sorry for the noise.. :)